backend/src/ee/services/pam-session/pam-session-dal.ts

```diff
@@ -22,5 +24,19 @@ export const pamSessionDALFactory = (db: TDbClient) => {
     return session;
   };
 
-  return { ...orm, findById };
+  const expireSessionById = async (sessionId: string, tx?: Knex) => {
+    const now = new Date();
+
+    const updatedCount = await (tx || db)(TableName.PamSession)
+      .where("id", sessionId)
+      .whereIn("status", [PamSessionStatus.Active, PamSessionStatus.Starting])
+      .update({
+        status: PamSessionStatus.Expired,
+        endedAt: now
+      });
```

Comment:
why do we use EXPIRED btw... can't we just make this "ended"? 

I think we shouldnt have a lot of status values so that its less overwhelming for users...